src/stories/containers/users/users-manager/users-manager.tsx

```diff
@@ -9,13 +9,14 @@ import lightTheme from '../../../../../styles/theme/light';
 import { useAuthContext } from '../../../../core/context/AuthContext';
 import { useThemeContext } from '../../../../core/context/ThemeContext';
 import { ButtonType } from '../../../../core/enums/button-type.enum';
-import { capitalizeWord } from '../../../../core/utils/string.utils';
+import { capitalizeWordWithoutConvertLowerCase } from '../../../../core/utils/string.utils';
 import { CustomButton } from '../../../components/custom-button/custom-button';
 import { SearchInput } from '../../../components/search-input/search-input';
 import { Tabs } from '../../../components/tabs/tabs';
 import UserCard from '../../../components/user-card/user-card';
+import { ParenthesisNumber } from '../../transparency-report/transparency-report';
 import UserProfile from '../user-profile/user-profile';
-import { useManagerAccountViewModel } from './manager-account.mvvm';
+import { TRANSPARENCY_IDS, useManagerAccountViewModel } from './manager-account.mvvm';
```

Comment:
This variable name should be changed as it is not related to "transparency" stuffs

Reply:
you are right, already done